nemo/core/classes/exportable.py

```diff
@@ -126,18 +127,34 @@ def disabled_deployment_output_names(self):
         return set()
 
     @staticmethod
-    def __extract_dynamic_axes(port_name, ntype: NeuralType, dynamic_axes: defaultdict):
```

Comment:
What is convention about leading (and double) underscores in Nemo?

Reply:
made it _ to be consistent with _prepare_for_deployment. Should be consistent with Python/Pytorch

Reply:
Not sure what consistency we are talking about. Normal method should have no underscores. In-place op (on a tensor) should have a trailing one. I suggested to follow that on prepare_for_deployment to give a hint it may modify the internal graph quite a bit. Bit this method is not like that.